pkg/constants/constants.go

```diff
@@ -4,7 +4,7 @@ import "os"
 
 var (
 	// DefaultVClusterVersion is the default version of the virtual cluster to use
-	DefaultVClusterVersion = "0.13.0"
+	DefaultVClusterVersion = "v0.16.4"
```

Comment:
Please check if Hubble needs to send updated version/tag for vcluster

Reply:
@shubhamrajvanshi currently Hubble doesn't send any version details. That's part of the problem. Once Hubble starts configuring the vcluster version properly we won't need to change these defaults. See the notes in the description for https://spectrocloud.atlassian.net/jira/software/c/projects/PPD/issues/PPD-1395.

Reply:
Also, @ahmed-kadri-tt adding the `v` here just results in the controller having to remove it. Would be better to leave it off.

Reply:
Ok, I thought that was taken care as part of support for multiple distributions/flavors